services-camel-app/src/main/java/edu/unc/lib/boxc/services/camel/views/ViewBehaviorRequestProcessor.java

```diff
@@ -0,0 +1,60 @@
+package edu.unc.lib.boxc.services.camel.views;
+
+import edu.unc.lib.boxc.auth.api.Permission;
+import edu.unc.lib.boxc.auth.api.services.AccessControlService;
+import edu.unc.lib.boxc.model.api.objects.RepositoryObjectLoader;
+import edu.unc.lib.boxc.model.api.objects.WorkObject;
+import edu.unc.lib.boxc.model.api.rdf.CdrView;
+import edu.unc.lib.boxc.model.api.services.RepositoryObjectFactory;
+import edu.unc.lib.boxc.model.fcrepo.ids.PIDs;
+import edu.unc.lib.boxc.operations.jms.views.ViewBehaviorRequestSerializationHelper;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+
+import java.util.Objects;
+
+/**
+ * Processor for requests for updating the view behavior for UV
+ *
+ * @author snluong
+ */
+public class ViewBehaviorRequestProcessor implements Processor {
```

Comment:
Just a suggestion, but it might make sense to name the various classes `ViewSettings` rather than Behavior. `Settings` is the terminology used in the front end code, and this same workflow will likely be used for setting other view settings, which behavior is one of those settings.

Similarly, maybe it'd be good to also have the package for these classes be `edu.unc.lib.boxc.services.camel.viewSettings`? `views` by itself makes me think it would be front end code